timeseries/src/autogluon/timeseries/predictor.py

```diff
@@ -23,54 +23,63 @@
+        - ``"sMAPE"`` - symmetric mean absolute percentage error
+        - ``"MASE"`` - mean absolute scaled error
 
-        If ``eval_metric is None``, it is set by default to "mean_wQuantileLoss".
+        # TODO: Evaluator is not part of the API, do we expect people to dig into the code? Maybe include description
+        # of metrics in the tutorial?
         For more information about these options, see ``autogluon.timeseries.TimeSeriesEvaluator`` and GluonTS
-        docs at https://ts.gluon.ai/api/gluonts/gluonts.evaluation.metrics.html
-    path : str, default = None
-        Path to directory where models and intermediate outputs should be saved. If unspecified, a timestamped folder
+        docs at https://ts.gluon.ai/stable/api/gluonts/gluonts.evaluation.metrics.html
+    quantile_levels : List[float] or None, default = None
+        List of increasing decimals that specifies which quantiles should be estimated when making distributional
+        forecasts. If None, defaults to ``[0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9]``.
+        Can alternatively be provided with the keyword argument ``quantiles``.
+        # TODO: Add link to explanation of distributional forecasts?
+    path : str or None, default = None
+        Path to the directory where models and intermediate outputs will be saved. If None, a timestamped folder
         ``AutogluonModels/ag-[TIMESTAMP]`` will be created in the working directory to store all models.
     verbosity : int, default = 2
         Verbosity levels range from 0 to 4 and control how much information is printed to stdout. Higher levels
         correspond to more detailed print statements, and ``verbosity=0`` suppresses output including warnings.
         If using ``logging``, you can alternatively control amount of information printed via ``logger.setLevel(L)``,
-        where ``L`` ranges from 0 to 50 (Note: higher values of ``L`` correspond to fewer print statements,
-        opposite of verbosity levels).
-    prediction_length : int, default = 1
-        The forecast horizon, i.e., How many time points into the future forecasters should be trained to predict.
-        For example, if time series contain daily observations, setting ``prediction_length=3`` will train
-        models that predict up to 3 days in the future from the most recent observation.
-    quantile_levels : List[float], default = None
-        List of increasing decimals that specifies which quantiles should be estimated
-        when making distributional forecasts. Can alternatively be provided with the keyword
-        argument ``quantiles``. If ``None``, defaults to ``[0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9]``.
+        where ``L`` ranges from 0 to 50 (Note: higher values of ``L`` correspond to fewer print statements, opposite
+        of verbosity levels).
+
+    # TODO: Why not move Other Parameters to Parameters? At least `ignore_time_index` and `validation_splitter` seem relevant
```

Comment:
I think convention is to have `**kwargs` in Other Parameters. If you mean we can move ignore_time_index and validation_splitter to named keyword arguments, I agree. (Needless nit: as long as their default values are also specified in the constructor definition).